openduck-py/openduck_py/routers/voice.py

```diff
@@ -129,10 +130,19 @@ def interrupt(self, task: asyncio.Task):
     async def start_response(
         self,
         websocket: WebSocket,
-        db: AsyncSession,
         audio_data: np.ndarray,
     ):
         print("starting response")
+
+
+        async_engine = create_async_engine(async_connection_string)
+        SessionAsync = sessionmaker(
+            bind=async_engine,
+            expire_on_commit=False,
+            class_=AsyncSession,
+        )
+        db = SessionAsync()
```

Comment:
I believe you should be able to import this: https://github.com/uberduck-ai/openduck/blob/main/openduck-py/openduck_py/db/__init__.py#L51

Then, we need to make sure the connection is closed even if there are exceptions raised. Do that like this:

```python
with SessionAsync() as db:
    # code that uses db
```